tests/test_edit_area.py

```diff
@@ -0,0 +1,91 @@
+# A simple test script to add a new area to test .metabrainz.org
+
+from musicbrainz_bot.editing import MusicBrainzClient
+import musicbrainz_bot.config as cfg
+import mechanize
+import pytest
+
+
+@pytest.fixture(scope="function")
+def mb_client():
+    mb = MusicBrainzClient(
+        cfg.MB_USERNAME, cfg.MB_PASSWORD, cfg.MB_SITE, use_test_db=True
+    )
+    return mb
+
+
+@pytest.fixture(scope="function")
+def browser():
+    browser = mechanize.Browser()
+    browser.set_handle_robots(False)
+    browser.set_debug_redirects(False)
+    browser.set_debug_http(False)
+    return browser
+
+
+@pytest.fixture(scope="function")
+def area_updatable():
+    return {
+        "name": "test_area",
+        "comment": "disambiguation_comment",
+        "type": "3",
+        "iso_3166_1": ["YY"],
+        "iso_3166_2": ["YY-A"],
+        "iso_3166_3": ["YYYA"],
+        "url": [
+            {
+                "text": "https://www.wikidata.org/wiki/Q152",
+                "link_type_id": 358,
+            },
+        ],
+    }
+
+
+@pytest.fixture(scope="function")
+def area_update():
+    return {
+        "name": "test_area_EDITED",
+        "comment": "disambiguation_comment_EDITED",
+        "type": "4",
+        "iso_3166_1": ["YY"],
+        "iso_3166_2": ["YY-B"],
+        "iso_3166_3": ["YYYB"],
+        "url": [
+            {
+                "text": "https://www.wikidata.org/wiki/Q153",
+                "link_type_id": 358,
+            },
+        ],
+    }
+
+
+def _add_area(area, mb, browser):
+    edit_note = "Tests new area with name, type, disambiguation, ISO 3166-1, ISO 3166-2, ISO 3166-3, URL, edit note."
+    area_mbid = mb.add_area(area, edit_note=edit_note)
+
+    return area_mbid
+
+
+def _edit_area(area, update, gid, mb, browser):
+    edit_note = "Tests new area with name, type, disambiguation, ISO 3166-1, ISO 3166-2, ISO 3166-3, URL, edit note."
+    area_mbid = mb.edit_area(gid, area, update, edit_note=edit_note)
+
+    return area_mbid
+
+
+def test_edit_area(mb_client, browser, reset_db, area_updatable, area_update):
```

Comment:
This does not test any of the values were updated, does it?

Reply:
On it :)

Reply:
Done! Except for a tiny issue, that I am writing in another comment on this PR thread.